packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -3916,6 +3976,10 @@ export class ProjectService extends BaseService {
             },
             async () => {
                 const spaces = await this.spaceModel.find({ projectUuid }); // Get all spaces in the project
+
+                // TODO we do not need to filter allowed space content here,
```

Comment:
Since we don't need, should we remove? Or should this be raised as a separate issue?

Reply:
I was going to leave it commeted out, but we can always check the code in the git history.